lambda/src/main/resources/db/migration/V19__remove_duplicate_consignment_properties.sql

```diff
@@ -0,0 +1,7 @@
+-- V17 and V18 migrations both add properties named: 'FinalOpenRecordsConfirmed' and 'LegalOwnershipTransferConfirmed'
+-- remove the properties and re-add them with new uuids as not clear which version of the properties should be used
+DELETE FROM "ConsignmentProperty" WHERE "Name" = 'FinalOpenRecordsConfirmed' OR "Name" = 'LegalOwnershipTransferConfirmed';
+
+INSERT INTO "ConsignmentProperty" ("PropertyId", "Name", "Description", "Shortname")
+VALUES (uuid_generate_v4(), 'FinalOpenRecordsConfirmed', '', ''),
```

Comment:
👍 

It would be good to add another migration (either in this PR or separately) which adds a uniqueness constraint to the Name property. That would stop this from happening again.

The same goes for the FileProperty table.

Reply:
I'll add another migration to do this, and raise it as a separate PR.